### PR TITLE
Issue 1: Modified PayPal plugin per PayPal's new requirements https://devblog.paypal.com/upcoming-security-changes-notice/  Issue 2: Rreturn order id along with error in payment detail, Added 2nd parameter to function onTP_Processpayment(),  phpcs

### PR DESCRIPTION
**Pull request by vidyasagarmundhe**
- In  a case of error while payment or on canceling the transaction,  the plugin doesn't return actual order id. 
-  Added second parameter(vars) to function onTP_Processpayment() which should be there according to new plugin structure. (Already present in all other plugins)
-  PHPCS

### CODE REVIEW

**Comment by coolbung** [comment]
Can you elaborate on the problem that we're trying to solve ?

File: paypal/paypal/helper.php [hunk 1]
```diff
@@ -53,6 +53,11 @@ public function Storelog($name, $logdata)
 		$options = "{DATE}\t{TIME}\t{USER}\t{DESC}";
 		$my      = JFactory::getUser();
 
+		if (empty($logdata['JT_CLIENT']))
```
Comment (coolbung) [inline paypal/paypal/helper.php]: What is JT_CLIENT ?

**Comment by vidyasagarmundhe** [comment]
Modified PayPal plugin per PayPal's new requirements https://devblog.paypal.com/upcoming-security-changes-notice/

File: paypal/paypal/helper.php (hunk 1 above)
Comment (vidyasagarmundhe) [inline paypal/paypal/helper.php]: It client : who is using the plugin. Eg, com_quick2cart, com_jticketing etc
it is used to create log file name in Joomla's log folder.

Comment (coolbung) [inline paypal/paypal/helper.php]: Ok by why does the variable name contain JT ?

Comment (vidyasagarmundhe) [inline paypal/paypal/helper.php]: Old naming is wrong. 
Ideally, it should be like CPG_CLIENT OR CLIENT

**Pull request merged**